ios/Classes/SwiftHmssdkFlutterPlugin.swift

```diff
@@ -916,14 +917,32 @@ public class SwiftHmssdkFlutterPlugin: NSObject, FlutterPlugin, HMSUpdateListene
     }
 
     
-    private func stopHlsStreaming(_ result: @escaping FlutterResult){
+    private func stopHlsStreaming(_ call: FlutterMethodCall,_ result: @escaping FlutterResult){
+        if let arguments = call.arguments as? [AnyHashable: Any],
+              let meetingUrlVariantsList = arguments["meeting_url_variants"] as? [[String:String]] {
```

Comment:
create a function that returns `HMSHLSConfig` or nil. Parameter input will be `arguments`
like so -
`let config = getHLSConfig(from: arguments)`

then only single usage of `hmsSDK?.stopHLSStreaming` will be required